de.fhdo.lemma.eclipse.ui/META-INF/MANIFEST.MF

```diff
@@ -20,7 +20,8 @@ Require-Bundle: org.eclipse.ui,
  de.fhdo.lemma.utils,
  de.fhdo.lemma.technology.mappingdsl.metamodel,
  de.fhdo.lemma.eclipse.ui.utils,
- de.fhdo.lemma.operationdsl.metamodel
+ de.fhdo.lemma.operationdsl.metamodel,
+ de.fhdo.lemma.service.open_api;bundle-version="0.7.0",
+ org.slf4j.api;bundle-version="1.7.2"
 Automatic-Module-Name: de.fhdo.lemma.eclipse.ui
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Import-Package: org.slf4j;version="1.7.2"
```

Comment:
Is it necessary to have SLF4J a dependency instead of "just" an imported OSGi package (as it was before)?